src/components/common/Bar.vue

```diff
@@ -2,13 +2,10 @@
   <div v-if="showMessage" :class="`bar ${barType}`">
     <p>
       <slot />
-      <span v-if="link" class="link" @click="goToLink(link)">{{
-        linkCaption
-      }}</span>
     </p>
     <div class="right">
       <button v-if="link" class="button" @click="goToLink(link)">
```

Comment:
```suggestion
      <button v-if="link" class="button small" @click="goToLink(link)">
```

Reply:
This doesn't change anything 🤔 What styles do you want me to add specifically?

Reply:
@jbibla that was your spec

Reply:
why doesn't it change anything?

Reply:
you're not using TmBtn....?

Reply:
you have to use TmBtn it looks like all our other buttons